src/properties.h

```diff
@@ -20,3 +20,7 @@
 
 int properties_create_device(struct knot_thing *thing,
 			     struct device_settings *conf_files);
+
+int properties_clear_credentials(struct knot_thing *thing, char *filename);
+int properties_store_credentials(struct knot_thing *thing, char *filename,
+				 char *id, char *token);
```

Comment:
Can you check if this column alignment is correct in your text editor?

Reply:
It is correct in the editor, it must be a GitHub bug.